src/databricks/labs/lakebridge/deployment/job.py

```diff
@@ -157,3 +162,124 @@ def parse_package_name(self, wheel_path: str) -> str:
+                del self._install_state.jobs[name]
+                logger.warning(f"Job `{name}` does not exist anymore for some reason")
+                return self._update_or_create_profiler_ingestion_job(
+                    name, catalog_name, schema_name, volume_location, source_tech, lakebridge_wheel_path
+                )
+
+        logger.info(f"Creating new job configuration for job `{name}`")
+        new_job = self._ws.jobs.create(**job_settings)
+        assert new_job.job_id is not None
+        self._install_state.jobs[name] = str(new_job.job_id)
+        return str(new_job.job_id)
+
+    def _profiler_ingestion_job_settings(
+        self,
+        job_name: str,
+        task_key: str,
+        description: str,
+        catalog_name: str,
+        schema_name: str,
+        volume_location: str,
+        source_tech: str,
+        lakebridge_wheel_path: str,
+    ) -> dict[str, Any]:
+
+        latest_lts_spark = self._ws.clusters.select_spark_version(latest=True, long_term_support=True)
+        version = self._product_info.version()
+        version = version if not self._ws.config.is_gcp else version.replace("+", "-")
+        tags = {"version": f"v{version}"}
+        if self._is_testing():
+            # Add RemoveAfter tag for test job cleanup
+            date_to_remove = self._get_test_purge_time()
+            tags.update({"RemoveAfter": date_to_remove})
+
+        return {
+            "name": self._name_with_prefix(job_name),
+            "tags": tags,
+            "job_clusters": [
+                JobCluster(
+                    job_cluster_key="Lakebridge_Profiler_Ingest_Cluster",
+                    new_cluster=compute.ClusterSpec(
+                        data_security_mode=compute.DataSecurityMode.USER_ISOLATION,
+                        spark_conf={},
+                        node_type_id=self._get_default_node_type_id(),
+                        autoscale=compute.AutoScale(min_workers=1, max_workers=3),
+                        spark_version=latest_lts_spark,
+                    ),
+                )
+            ],
+            "tasks": [
+                self._job_profiler_ingestion_task(
+                    task_key,
+                    description,
+                    lakebridge_wheel_path,
+                ),
+            ],
+            "max_concurrent_runs": 1,
+            "parameters": [
+                JobParameterDefinition(name="catalog_name", default=catalog_name),
+                JobParameterDefinition(name="schema_name", default=schema_name),
+                JobParameterDefinition(name="volume_path", default=volume_location),
+                JobParameterDefinition(name="source_tech", default=source_tech),
+            ],
+        }
+
+    def _job_profiler_ingestion_task(self, task_key: str, description: str, lakebridge_wheel_path: str) -> Task:
+        libraries = [
+            compute.Library(whl=lakebridge_wheel_path),
+            compute.PythonPyPiLibrary(package="duckdb")
```

Comment:
The ingestion job is dependent on `duckdb` library to read the profiler extract tables.